Gems/Atom/Feature/Common/Assets/Shaders/Materials/MaterialInputs/SilhouetteInput.azsli

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+option enum class SilhouetteType { AlwaysDraw, Visible, XRay, NeverDraw } o_silhouetteType = SilhouetteType::AlwaysDraw;
```

Comment:
Same. Where is this shader option used?

Reply:
This is the same situation where we are make an "unused" shader type just for the material property group settings.  This one is used in SilhouettePropertyGroup.json